hs_rest_api/resources/file_metadata.py

```diff
@@ -141,14 +141,16 @@ def put(self, request, pk, file_id):
 
             spatial_coverage = file_serializer.data.pop("spatial_coverage", None)
             if spatial_coverage is not None:
+                # defaulting to point if not provided for backwards compatibility
+                type = spatial_coverage["type"] if "type" in spatial_coverage else "point"
```

Comment:
pretty minor I think you can express this as:

`type = spatial_coverage.get("type", "point")`

Reply:
Unless you decide you feel strongly about it, I'm going to leave it for the sake of time.